app/app.go~HEAD

```diff
@@ -0,0 +1,245 @@
+package app
```

Comment:
this file shouldn't exist. there are also a ton of commits in the commit history here that are duplicated from other PRs. you sure conflict resolution worked out properly?

Reply:
@musnit that file keeps getting quietly added during merges. Removed in https://github.com/cosmos/peggy/pull/82/commits/160bd8ba89149433d30891d6130a1389aee6ddd6.